app/models/king.rb

```diff
@@ -1,12 +1,10 @@
 class King < Piece
-
-	# Should determine if the king piece can move to the designated spot
-	def valid_move?(destination_row, destination_col)
-		# Call obstructed method to see if there is already a piece in the
-		# destination_row/destination_column
-		!obstructed?(destination_row, destination_col) && distance(destination_row, destination_col) == 1
-	end
+	 # Should determine if the king piece can move to the designated spot
+	 def valid_move?(destination_row, destination_col)
+ 		 # Call obstructed method to see if there is already a piece in the
+ 		 # destination_row/destination_column
+ 		 !obstructed?(destination_row, destination_col) && distance(destination_row, destination_col) == 1
```

Comment:
In the card I wrote to use `obstructed`. But I'm thinking now that was a mistake, sorry! :)

For valid_move, if the destination square has a piece of the `opposite` color, then it is a valid move. Does this code cover that?

`obstructed` only checks for pieces between the start and destination. Since in this case there aren't any. I don't think it's really needed (we will need it for other pieces valid_moves though).